docker/.env.ik

```diff
@@ -0,0 +1,42 @@
+# Project name
+COMPOSE_PROJECT_NAME=smartgear
+COMPOSE_PROJECT_NAME_SHORT=sm
+PRJ_NAME='smartgear'
+BUNDLE_NAME='contao-smartgear'
+
+# Database settings
+DB_HOST='db' # database service name in docker-compose file
+DB_DATABASE='ms_test'
+DB_USER='smartgear'
+DB_PASSWORD='root'
+DB_ROOT_PASSWORD='root'
+# DATABASE_URL="mysql://${DB_USER}:${DB_PASSWORD}@${DB_HOST}:3306/${DB_DATABASE}" #Yes, 3306, because inside docker network, the internal ports are useable, not the exposed ones
+
+# Ports settings
+PHP_FPM_EXPOSED_PORT=9002
+DB_EXPOSED_PORT=3309
+WEBSERVER_EXPOSED_PORT=8004
+WEBSERVER_EXPOSED_PORT_SSL=8005
+
+PHP_FPM_EXPOSED_PORT_TEST=9003
+DB_EXPOSED_PORT_TEST=3310
+WEBSERVER_EXPOSED_PORT_TEST=8006
+WEBSERVER_EXPOSED_PORT_SSL_TEST=8007
+
+# Directories settings
+WORKDIR_BASE='/usr/local/apache2/htdocs'
+WORKDIR_INIT="${WORKDIR_BASE}/init"
+WORKDIR_CONTAO="${WORKDIR_BASE}/${PRJ_NAME}"
+WORKDIR_CONTAO_PUBLIC="${WORKDIR_CONTAO}/public"
+# WORKDIR_BUNDLE="${WORKDIR_BASE}/${BUNDLE_NAME}"
+WORKDIR_BUNDLE="${WORKDIR_BASE}/${PRJ_NAME}/vendor/webexmachina/${BUNDLE_NAME}"
+WORKDIR_SIMLINK_BASE="${WORKDIR_BASE}/${PRJ_NAME}/vendor/webexmachina"
+WORKDIR_BUNDLE_END_SLASH="${WORKDIR_BUNDLE}/"
+
+# PHP settings (those are our main host default values)
+PHP_INI_MAX_EXECUTION_TIME=60
```

Comment:
@LupusVII I reckon I got all our default host's PHP limitations here. Could you check if anything 's missing or has a wrong value ?

Reply:
It should work with a 60s max execution time so ok for me.